### PR TITLE
[Documentation] Fixing ambiguous language in network identity section.

### PR DESCRIPTION
**Pull request by rajkaramchedu**
Motivated by this PR: https://github.com/aptos-labs/aptos-core/pull/866, I am making a few clarifications in the network identity section of the fullnode tutorial doc:

Note: The Docusaurus preview shows proper formatting. Hence I am not fixing the code ticks issue we see here in the GitHub preview. If you insist, I will fix. Thanks!

cc: @gregnazario

### CODE REVIEW

File: developer-docs-site/docs/tutorials/full-node/network-identity-fullnode.md [hunk 1]
```diff
@@ -6,10 +6,20 @@ sidebar_position: 12
...
 # Network Identity For FullNode
 
+FullNodes will automatically start up with a randomly generated network identity. This works well for regular FullNodes. However:
+
+- You may want your FullNode to be added to a specific upstream FullNode's allowlist (i.e., other FullNode participant in the Aptos network), because:
+
+  - You might require specific permissions for your FullNode on this specific upstream FullNode, or
+  - This upstream FullNode only allows known identities to connect to it.
+
+In such cases, it helps if you run your FullNode with the same (static) network identity, instead of a randomly generated network identity that keeps changing every time you start up your FullNode.
```
Comment (davidiw) [inline developer-docs-site/docs/tutorials/full-node/network-identity-fullnode.md:16]:
```suggestion
In such cases, it helps if you run your FullNode with a static network identity, instead of a randomly generated network identity that keeps changing every time you start up your FullNode.
```

File: developer-docs-site/docs/tutorials/full-node/network-identity-fullnode.md [hunk 2]
```diff
@@ -23,7 +33,7 @@ Docker container is currently supported only on Linux x86-64 platform. If you ar
 
 ## Creating a static identity for a FullNode
 
-FullNodes will automatically start up with a randomly generated network identity (a `PeerId` and a public key pair). This works well for regular FullNodes, but you may wish to be added to another node's allowlist, provide specific permissions or run your FullNode with the same identity. In this case, creating a static network identity can help.
+To create a static identity for your FullNode, you should first create a private key public key pair for your FullNode. You then use the public key in the `peer_id` field in your `public_full_node.yaml` to create a static network identity for your FullNode.
```
Comment (davidiw) [inline developer-docs-site/docs/tutorials/full-node/network-identity-fullnode.md:36]:
```suggestion
To create a static identity for your FullNode, you should first create a private key, public key pair for your FullNode. You then use the public key in the `peer_id` field in your `public_full_node.yaml` to create a static network identity for your FullNode.
```

> You then use the public key in the `peer_id` field in your `public_full_node.yaml` to create a static network identity for your FullNode.

This isn't correct, the peer_id is derived from the public key.

File: developer-docs-site/docs/tutorials/full-node/network-identity-fullnode.md [hunk 3]
```diff
@@ -51,23 +63,12 @@ FullNodes will automatically start up with a randomly generated network identity
...
-    ```
-    aptos key generate --key-type x25519 --output-file /path/to/private-key.txt
-
-    ```
-
-    **Using Docker**
-
-    From inside the `aptoslab/tools` Docker container. Open a new terminal and `cd` into the directory where you started the Docker container for your FullNode. Making sure to provide the full path to where you want the private key TXT file to be stored, run the command as below:
+  ```
+  aptos key generate --key-type x25519 --output-file /path/to/private-key.txt
```
Comment (davidiw) [inline developer-docs-site/docs/tutorials/full-node/network-identity-fullnode.md:68]: nit unnecessary whitespace

File: developer-docs-site/docs/tutorials/full-node/network-identity-fullnode.md [hunk 4]
```diff
@@ -123,25 +137,28 @@ full_node_networks:
...
 ## Allowing other FullNodes to connect
 
-Once you start your FullNode with a static identity you can allow others to connect to devnet through your node. Follow these recommendations:
+Once you start your FullNode with a static identity you can allow others to connect to devnet through your node. Follow these steps:
 
 - Make sure you open port `6180` (or `6182`, depending on which port your node is listening to) and that you open your firewall.
 - If you are using Docker, simply add `- "6180:6180"` or `- "6182:6182"` under ports in your ``docker-compose.yaml`` file.
-- You'll need to share your FullNode info for others to use as `seeds` in their configurations (e.g., `peer-info.yaml`):
+- Share your FullNode static network identity with others. They can then use it in the `seeds` key of their `public_full_node.yaml` file to connect to your FullNode.
+- Make sure the port number you put in the `addressses` matches the one you have in the FullNode config (`6180` or `6182`).
```
Comment (davidiw) [inline developer-docs-site/docs/tutorials/full-node/network-identity-fullnode.md:159]: 6180 / 6182 are optional, the port can be selected by the user.

Comment (davidiw) [reply developer-docs-site/docs/tutorials/full-node/network-identity-fullnode.md:159]: maybe for example or e.g.

File: developer-docs-site/docs/tutorials/full-node/network-identity-fullnode.md (hunk 2 above)
Comment (rajkaramchedu) [reply developer-docs-site/docs/tutorials/full-node/network-identity-fullnode.md:36]: Thanks for catching this. I also added an additional command using Aptos CLI showing how to extract peer. Sorry about this churn.

File: developer-docs-site/docs/tutorials/full-node/network-identity-fullnode.md [hunk 5]
```diff
@@ -6,10 +6,20 @@ sidebar_position: 12
 
 # Network Identity For FullNode
 
+FullNodes will automatically start up with a randomly generated network identity. This works well for regular FullNodes. However:
+
+- You may want your FullNode to be added to a specific upstream FullNode's allowlist (i.e., other FullNode participant in the Aptos network), because:
```
Comment (JoshLind) [inline developer-docs-site/docs/tutorials/full-node/network-identity-fullnode.md:11]: nit: "(i.e., other..." -> "(i.e., another..."

File: developer-docs-site/docs/tutorials/full-node/network-identity-fullnode.md [hunk 6]
```diff
@@ -6,10 +6,20 @@ sidebar_position: 12
 
 # Network Identity For FullNode
 
+FullNodes will automatically start up with a randomly generated network identity. This works well for regular FullNodes. However:
```
Comment (JoshLind) [inline developer-docs-site/docs/tutorials/full-node/network-identity-fullnode.md:9]: nit: maybe add another example use-case, e.g.,
```
- You may wish to advertise your FullNode for other Aptos FullNodes to connect to (to help support the network).
```

File: developer-docs-site/docs/tutorials/full-node/network-identity-fullnode.md [hunk 7]
```diff
@@ -6,10 +6,20 @@ sidebar_position: 12
...
+
+In such cases, it helps if you run your FullNode with a static network identity, instead of a randomly generated network identity that keeps changing every time you start up your FullNode.
+
 This guide will show you how to:
+
 - Create a static network identity for your FullNode.
-- Retrieve the public network identity.
-- Start a node with (or without) a static network identity.
+- Start a node with a static network identity.
+- Allow other FullNodes to connect to your Fullnode.
```
Comment (JoshLind) [inline developer-docs-site/docs/tutorials/full-node/network-identity-fullnode.md:22]: nit: "Fullnode" -> "FullNode".

I really wish we hadn't normalized "FullNode" 😵‍💫

File: developer-docs-site/docs/tutorials/full-node/network-identity-fullnode.md [hunk 8]
```diff
@@ -36,7 +50,9 @@ FullNodes will automatically start up with a randomly generated network identity
 
     **Using Docker**
 
-    Alternatively, if you are on Linux x86-64 platform, you can use Aptos Docker image. `cd` into the directory for your local public FullNode and start a Docker container with the latest tools, for example:
+    Alternatively, if you are on Linux x86-64 platform, you can use Aptos Docker image.
```
Comment (JoshLind) [inline developer-docs-site/docs/tutorials/full-node/network-identity-fullnode.md:53]: nit: use *the* Aptos Docker image.

**Review by JoshLind — COMMENTED** [review]
Thanks @rajkaramchedu! A couple nits. Otherwise, looks good 😄

**Review by JoshLind — APPROVED** [review]
Also, unblocking this 😄
